hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacility.java

```diff
@@ -77,13 +76,9 @@ public SystemFileUpdateFacility(
      * @param state the current state (the updated file content needs to be committed to the state)
      * @param txBody the transaction body
      */
-    public void handleTxBody(
-            @NonNull final HederaState state,
-            @NonNull final TransactionBody txBody,
-            @NonNull final SingleTransactionRecordBuilderImpl recordBuilder) {
+    public void handleTxBody(@NonNull final HederaState state, @NonNull final TransactionBody txBody) {
```

Comment:
It looks like Valentin just added the SingleTransactionRecordBuilderImpl argument in PR #8219 that was merged three weeks ago. I wonder why he added it, as you noticed it is not actually being used.

Reply:
As far as I can remember I had to do some refactoring and this field was a miss. Good catch on cleaning that up 🏆

Reply:
Yes, it's not used anywhere.